index.bs

```diff
@@ -1310,11 +1316,10 @@ To <dfn>trigger attribution</dfn> given an [=attribution trigger=] |trigger| run
 
 <h3 algorithm id="delivery-time">Establishing report delivery time</h3>
 
-To <dfn>obtain an expiry deadline</dfn> given an [=attribution source=]
+To <dfn>obtain an event report window deadline</dfn> given an [=attribution source=]
 |source|:
 
-1. Return the maximum of |source|'s [=attribution source/expiry=] and 2 days
-    (172800 seconds).
+1. Return |source|'s [=attribution source/event report window=].
```

Comment:
Is it still worth keeping this definition around if it's trivially returning the field?